cmake/scripts/android/Install.cmake

```diff
@@ -153,6 +153,7 @@ foreach(lib IN LISTS required_dyload dyload_optional ITEMS Shairplay)
 endforeach()
 add_bundle_file(${ASS_LIBRARY} ${libdir} "")
 add_bundle_file(${BLURAY_LIBRARY} ${libdir} "")
+add_bundle_file(${NFS_LIBRARY} ${libdir} "")
```

Comment:
since we link static, why do we need this?

Reply:
see PR description
> Don't load libnfs dynamic, but link against it (shared/static).

Currently libnfs is build shared in depends.

Reply:
I have both a dynamic and static lib, so we build both. In theory cmake should prefer and link the static lib. If not, we should fix this and not even build the dynamic one

Reply:
I planed to do the depends changes in a seperate PR. Should I do them in this PR @wsnipex?

Reply:
yes please

Reply:
@wsnipex updated